samples/bluetooth/mesh/light_switch/prj.conf

```diff
@@ -19,7 +19,6 @@ CONFIG_DK_LIBRARY=y
 CONFIG_BT=y
 CONFIG_BT_COMPANY_ID=0x0059
 CONFIG_BT_DEVICE_NAME="Mesh Light Switch"
-CONFIG_BT_L2CAP_RX_MTU=69
 CONFIG_BT_L2CAP_TX_MTU=69
```

Comment:
Setting of TX_MTU (for Mesh samples with proxy protocol enabled) could be remove as well. This should be handled by the default now.

Reply:
Sorry, this was meant as a general comment to all of the mesh/**/prj.conf that was touched. In all cases where RX_MTU was assigned to 69 TX_MTU could be removed as well.

It is not critical though, just more clean.